R/survival_ln_mixture-fit.R

```diff
@@ -33,10 +32,16 @@
 #'
 #' @param starting_seed Starting seed for the sampler. If not specified by the user, uses a random integer between 1 and 2^28 This way we ensure, when the user sets a seed in R, that this is passed into the C++ code.
 #'
-#' @param sparse Useful if the design matrix is sparse (most cases with categorical only regressors). Can save a lot of memory, allowing for huge data to be fitted.
-#'
 #' @param use_W Specifies is the W (groups weight's matrix for each observation) should be used from EM. It holds W constant through the code, resulting in a faster Bayesian Inference (close to what Empirical Bayes would do). It may helps generating credible intervals for the survival and hazard curves, using the information from the previous EM iteration. Make sure the EM have converged before setting this parameter to true. In doubt, leave this as FALSE, the default.
 #'
+#' @param better_initial_values A logical value indicating if the algorithm should search for better initial values of the EM algorithm. Recommended to leave it to TRUE, always, since the computational cost is very small.
```

Comment:
Victor, se não tem nenhum ponto negativo, será que não vale a pena remover esse parâmetro e usar como se fosse sempre TRUE de fato?

Reply:
Eu acabei deixando a parâmetro para efeitos de comparação, mas acho que podemos deixar como default um número baixo de EMs para rodar (atualmente rodam 200 EM com 1 passo cada um, basicamente na ideia de iniciar os parâmetros em vários valores aleatórios e ver qual início possui a maior verossimilhança) e daí podemos desabilitar utilizando o parâmetro **number_em_search = 0**.